doc/QC2-sample-example.mzQC.md

```diff
@@ -9,33 +9,33 @@ Optional (detailed) descriptions about the file can be placed into mzQC next to
 ```
 The metrics describe simple values like the cornerstone numbers of the acquisition and identification process, but also information specific to the QC method employed. Here, it is the mass accuracies and MS1 feature areas of selected peptides. With regular runs like this, the instrument's performance can be reliably monitored and maintenance interventions scheduled before valuable samples are wasted on an instrument running at sub-par performance. 
 ```
-          {
-            "accession": "QC:4000244",
-            "name": "QC2 sample mass accuracies",
-            "value": {
-              "MS:1003169": ["YAEAVTR","STLTDSLVC(Carbamidomethyl)K","SLADELALVDVLEDK","NPDDITNEEYGEFYK","LAVDEEENADNNTK","FEELNMDLFR","EAALSTALSEK","DDVAQTDLLQIDPNFGSK","RFPGYDSESK","EATTEFSVDAR","EQFLDGDGWTSR","TPAQFDADELR","LGDLYEEEMR","EVSTYIK","FAFQAEVNR"],
-              "QC:4000243": [-0.2346854518740762,-0.08024023890884578,-0.1322012562867409,-0.2259441806378488,-0.10596535779273217,0.28345130855013684,-0.08600783742175504,-0.3683484942567654,-0.03348194493295555,-0.41789282666789496,-0.12794363836212685,0.0,0.0,0.0,0.0]
-            }
-	  },
-          {
-            "accession": "QC:4000242",
-            "name": "QC2 sample intensities",
-            "value": {
-              "MS:1003169": ["YAEAVTR","STLTDSLVC(Carbamidomethyl)K","SLADELALVDVLEDK","NPDDITNEEYGEFYK","LAVDEEENADNNTK","FEELNMDLFR","EAALSTALSEK","DDVAQTDLLQIDPNFGSK","RFPGYDSESK","EATTEFSVDAR","EQFLDGDGWTSR","TPAQFDADELR","LGDLYEEEMR","EVSTYIK","FAFQAEVNR"],
-              "MS:1001844": [1234940000.0,922790000.0,80819100.0,478714000.0,254935000.0,52841200.0,243597000.0,24581800.0,707504000.0,129063000.0,205583000.0,0.0,0.0,0.0,0.0]
-            }
-          },
+               {
+                  "accession":"MS:4000078",
+                  "name":"QC2 sample mass accuracies",
+                  "value":{
+                     "MS:1003169":["YAEAVTR","STLTDSLVC(Carbamidomethyl)K","SLADELALVDVLEDK","NPDDITNEEYGEFYK","LAVDEEENADNNTK","FEELNMDLFR","EAALSTALSEK","DDVAQTDLLQIDPNFGSK","RFPGYDSESK","EATTEFSVDAR","EQFLDGDGWTSR","TPAQFDADELR","LGDLYEEEMR","EVSTYIK","FAFQAEVNR"],
+                     "MS:4000072":[-0.2346854518740762,-0.08024023890884578,-0.1322012562867409,-0.2259441806378488,-0.10596535779273217,0.28345130855013684,-0.08600783742175504,-0.3683484942567654,-0.03348194493295555,-0.41789282666789496,-0.12794363836212685,0.0,0.0,0.0,0.0]
```

Comment:
Are these values at the end really 0 ppm, or should they actually be removed (not detected, as mentioned in the CV term definition)?

Reply:
No, they are not detected (sample intensities also 0). Would this make the example more or less intelligible? I would tend to remove, but little hesitant.

Reply:
A 0 value is ambiguous, as demonstrated by my confusion. Does 0 mean that they're detected with perfect mass accuracy, or not detected? Unclear.

Also, based on the CV definition, if they're not detected they shouldn't be listed.

Reply:
I think it's a pretty good demonstration of the definition being spot on. Since 0 would be ambiguous, we defined one of the ambiguity cases to not be present.
I was just thinking if we should have an example of complete detection.

Reply:
I think it's fine to have missing values. That's just a reflection of real data.